pkgs/test_core/lib/src/util/io.dart

```diff
@@ -87,6 +87,12 @@ final _tempDir = Platform.environment.containsKey('_UNITTEST_TEMP_DIR')
 bool get canUseSpecialChars =>
     (!Platform.isWindows || stdout.supportsAnsiEscapes) && !inTestTests;
 
+/// Detect whether we're running in a Github Actions context.
+///
+/// See
+/// https://docs.github.com/en/actions/learn-github-actions/environment-variables.
+bool get githubContext => Platform.environment['GITHUB_ACTIONS'] == 'true';
```

Comment:
I could call this maybe `inGithubContext` or something, it doesn't sound like a boolean to me with the current name

Reply:
👍 will do